typings.d.ts

```diff
@@ -179,11 +205,17 @@ declare module 'resin-components' {
 		class h4 extends RenderableElementWithProps<FlexProps, any> {}
 	}
 
-	interface LinkProps
-		extends DefaultProps,
-			React.AnchorHTMLAttributes<HTMLElement> {
+	interface LinkProps extends DefaultProps {
 		blank?: boolean;
 		disabled?: boolean;
+		download?: any;
```

Comment:
I think this should be `string`, shouldn't it?

Reply:
The download attribute doesn't actually have to have a value https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-download
I guess this means that it could be a boolean.
This attribute is taken from the React typings so I would prefer to follow their leas.

Reply:
Just to clarify [it can have a string value](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#Using_the_download_attribute_to_save_a_<canvas>_as_a_PNG) that will be used as the suggested filename. Pretty NIT though (IMO) to discuss this further...